package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@polymarket/clob-client",
     "description": "Typescript client for Polymarket's CLOB",
-    "version": "1.5.1",
+    "version": "1.6.0",
```

Comment:
will likely end up being 2.1.0. There is another PR open (#80) with a published package that will roll out soon and it is breaking so it bumps the major version

Reply:
done